src/Chirp.Razor/CheepService.cs

```diff
@@ -51,7 +51,13 @@ public async Task Create(CheepCreateDTO cheep)
     }
     public async Task AddAuthor(string name, string email)
     {
-        await _authorRepository.AddAuthor(name, email);
+        _authorRepository.AddAuthor(name, email);
+    }   
+    public void DeleteCheepsFromAuthor(int authorid){
```

Comment:
We should change this method to be asynchronous.

I am happy to help if necessary.

Reply:
Which method are you refering to??

Reply:
The DeleteCheepsFromAuthor method